tooling/specs/specs.go

```diff
@@ -27,16 +27,42 @@ func (m maturity) isMature() bool {
 type Spec string
 
 const (
+	// Real HTTP Gateway Specs: https://specs.ipfs.tech/http-gateways/
+	PathGateway      Spec = "path-gateway"
+	TrustlessGateway Spec = "trustless-gateway"
+	DNSLinkGateway   Spec = "dnslink-gateway"
 	SubdomainGateway Spec = "subdomain-gateway"
-	DNSLinkResolver  Spec = "dnslink-resolver"
-	IPNSResolver     Spec = "ipns-resolver"
+	RedirectsFile    Spec = "redirects-file" // This one can run for path gateway, dnslink and subdomain gateways.
+
+	// TrustlessGateway sub-specifications.
+	TrustlessBlockGateway Spec = "trustless-block-gateway"
+	TrustlessCARGateway   Spec = "trustless-car-gateway"
+	TrustlessIPNSGateway  Spec = "trustless-ipns-gateway"
+
+	// Helper, smaller range specs.
+	IPNSResolver Spec = "ipns-resolver"
 )
 
 // All specs should be listed here.
 var specMaturity = map[Spec]maturity{
+	PathGateway:      stable,
+	TrustlessGateway: stable,
+	DNSLinkGateway:   stable,
 	SubdomainGateway: stable,
-	DNSLinkResolver:  stable,
-	IPNSResolver:     stable,
+	RedirectsFile:    stable,
+
+	IPNSResolver: stable,
+}
+
+// Groups of specifications that, if enabled or disabled, will enable or disable
+// all the respective specifications. The group specifications must not be used for
+// RunIfSpecsAreEnabled, only the sub-specifications.
```

Comment:
👍

Reply:
Maybe as you said before, we can make some distinction between Leafs and Groups in order to ensure that is forced.